chatkit-core/config/baseline.xml

```diff
@@ -2,6 +2,8 @@
 <SmellBaseline>
   <Blacklist></Blacklist>
   <Whitelist>
+    <ID>ForbiddenComment:ChatkitState.kt$// TODO: uncomment when reducers branch is merged into v2 and this branch caught up</ID>
```

Comment:
this needs to be removed before it's merged in :)

Reply:
done: 5a50a17f98e924af74f238ffe68429cd6f87eed7